website/docs/cloud-docs/projects/manage.mdx

```diff
@@ -38,10 +38,10 @@ To view your organization's projects:
 1. Click **+ New project**.
 1. Specify a name for the project. The name must be unique within the organization and can only include letters, numbers, inner spaces, hyphens, and underscores.
 1. Add a description for the project. This field is optional.
-1. Open the **Add key value tags** menu to add tags to your project. Tags are optional key-value pairs that you can use to organize projects. Any workspaces you create within the project inherit project tags. Refer to [Define project tags](#define-project-tags) for additional information.
-
-   Functionality associated with key-value tags is in beta. Refer [2024-09-12](/terraform/cloud-docs/changelog#2024-09-12) in the changelog for additional information.
+   
+    ~> **Adding project tags is in limited beta and unavailable for some users.** Skip to step 8 if your interface does not include elements for adding tags. Contact your HashiCorp representative for information about participating in the limited beta.  
```

Comment:
"private beta" is the approved release stage name.

Reply:
```suggestion
    ~> **Adding project tags is in private beta and unavailable for some users.** Skip to step 8 if your interface does not include elements for adding tags. Contact your HashiCorp representative for information about participating in the limited beta.  
```